dev/release/verify-release-candidate.sh

```diff
@@ -468,6 +469,16 @@ install_conda() {
   conda activate base
 }
 
+install_maven() {
```

Comment:
there is a util called `util_download_apache.sh` which already does this quite better, should we rather update the container script to reuse that script?

Reply:
We can use only https://www.apache.org/dyn/closer.lua . We don't need to use mirrors.

See also: https://infra.apache.org/release-download-pages.html#download-page